stingray/utils.py

```diff
@@ -233,18 +227,20 @@ def rebin_data(x, y, dx_new, yerr=None, method='sum', dx=None):
         raise ValueError("Method for summing or averaging not recognized. "
                          "Please enter either 'sum' or 'mean'.")
 
-    tseg = x[-1] - x[0] + dx_old
+    tseg = x[-1] - x[0] + dx_old[-1]
```

Comment:
Shouldn't this be something like `x[-1] + dx_old[-1] / 2 - (x[0] - dx_old[0] / 2)`?

Reply:
Well, that depends on how the data is defined. I remember asking at the time, and I made the note that the code assumes the points given in `x` are the left bin edges. This is consistent with the previous version of this function, so I kept it at the same definition.